docs/edge.rst

```diff
@@ -326,16 +331,115 @@ into the form.
 
 .. _edge-tools:
 
-Install CrateDB Edge using an on-premise Kubernetes distribution
-================================================================
+Install CrateDB Edge using third-party tools
+============================================
+
+You can combine CrateDB Edge with third-party tools for ease of use, such as
+managed Kubernetes providers and preconfigured Kubernetes distributions. In the
+former category, we have tested AWS, Azure, Digital Ocean, and GCP with the
+CrateDB Edge stack, and for the latter the distributions described below. In
+the walkthroughs below, we provide by way of example a guide for using
+`Digital Ocean`_ as a managed Kubernetes provider with CrateDB Edge, and guides
+for two of the most common Kubernetes distributions: `Microk8s`_ and `K3s`_.
```

Comment:
```suggestion
for two of the most common Kubernetes distributions: `microk8s`_ and `k3s`_.
```

Reply:
They write it as "MicroK8s" and "K3s" themselves so I would follow that actually